commands/commands_test.go

```diff
@@ -23,3 +25,13 @@ func requireSchemaConformance(t *testing.T, jsonBytes []byte, schemaName string)
 
 	require.Truef(t, result.Valid(), "Error schema validating: %s", string(jsonBytes))
 }
+
+func makeTestDaemonWithMinerAndStart(t *testing.T) *th.TestDaemon {
+	daemon := th.NewDaemon(
```

Comment:
We were doing this in a bunch of places and most of the time it was fine to just use the first of everything (TestAddresses, TestMiners, KeyFilePaths), so I abstracted it.